core/src/main/java/com/opentable/logging/ApplicationLogEvent.java

```diff
@@ -99,7 +99,7 @@ public String getServiceType()
 
     @Override
     public String getLoglov3Otl() {
-        return "msg-v1";
+        return event.getMDCPropertyMap().getOrDefault("@loglov3-otl-override", "msg-v1");
```

Comment:
@SegFaultAX Should we check if the key contains null or empty? That's .an couple lines of code, so I'm ambivalent